expressEngine/cms/user/error.js

```diff
@@ -7,4 +7,7 @@ export default {
   loginError: 'login failed please signUp first',
   passwordError: 'incorrect password',
   tokenError: 'token is not generated',
+  unableToUpdate: 'User not found to update',
+  emptyData: 'Please provide data to update',
```

Comment:
@adilsuccessive  emptyData should be placed in Common cms.

Reply:
Done